src/client/gpm_common.c

```diff
@@ -278,7 +278,7 @@ static int gpm_epoll_wait(struct gpm_ctx *gpmctx, uint32_t event_flags)
 {
     int ret;
     int epoll_ret;
-    struct epoll_event ev;
+    struct epoll_event ev = { 0 };
```

Comment:
And this one by 285?

Reply:
I think I have basically resolved differently a leak you(?) already solved but my master/main tree was not up to date, gonna drop my commit